architecture/admission-controllers.adoc

```diff
@@ -0,0 +1,26 @@
+[id="admission-controllers"]
+= Admission controllers
+include::modules/common-attributes.adoc[]
+:context: admission-controllers
+toc::[]
+
+// Concept modules
+include::modules/admission-controller-about.adoc[leveloffset=+1]
+
+include::modules/admission-controller-default.adoc[leveloffset=+1]
+
+include::modules/admission-webhook-about.adoc[leveloffset=+1]
+
+include::modules/admission-webhook-types.adoc[leveloffset=+1]
+
+// Procedure module
+include::modules/admission-webhook-create.adoc[leveloffset=+1]
+
+[id="admission-controller-additional-resources"]
+== Additional resources
+
+* xref:../networking/hardware_networks/configuring-sriov-operator.adoc#configuring-sriov-operator[Limiting custom network resources managed by the SR-IOV network device plug-in].
```

Comment:
Not critical, but I'd suggest removing the periods in this bulleted list because they aren't complete sentences. Or make them into complete sentences.

Reply:
Resolved.